packages/transport-webrtc/src/private-to-private/signaling-stream-handler.ts

```diff
@@ -92,12 +92,20 @@ export async function handleIncomingStream ({ peerConnection, stream, signal, co
     })
   } catch (err: any) {
     if (getConnectionState(peerConnection) !== 'connected') {
-      log.error('error while handling signaling stream from peer %a', connection.remoteAddr, err)
+      log.error(
+        'error while handling signaling stream from peer %a - %e',
+        connection.remoteAddr,
+        err
+      )
```

Comment:
i'm viewing this on github, but I don't feel like this one needed to be broken into multiple lines